processors/piifilterprocessor/filters/json/filter.go

```diff
@@ -37,6 +40,16 @@ func (f *jsonFilter) RedactAttribute(key string, value pdata.AttributeValue) (bo
 
 	err := f.mu.UnmarshalFromString(value.StringVal(), &jsonPayload)
 	if err != nil {
+		// if json is invalid, run the value filter on the json string to try and
+		// filter out any keywords out of the string
+		f.logger.Debug("Problem parsing json. Falling back to value regex filtering")
```

Comment:
should it log the payload? Is the logging level right?

Reply:
I think debug is the right log level, this isn't that actionable as a common use case is the limit in the size of the attribute (also this is data coming from the user). If this is `Debug` I am happy to add the payload in the log, otherwise (e.g. if info) I'd not include the payload as it would flood the logs.

Reply:
yeah, debug makes sense here - we could see a LOT of these messages.